src/client/components/Loader/Loader.js

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import './Loader.styles.css';
 
-export default function Loader({ loadingText }) {
-  return <div className="loading">{loadingText}</div>;
+export default function Loader() {
+  return (
+    <div>
```

Comment:
Is this outer `div` really necessary?

Reply:
No it actually wasn't. I removed it. Thanks